### DISCUSSION TITLE
真是特别棒的前端模板

### DISCUSSION THREAD

**Discussion by elarbor** [HalseySpicy/Geeker-Admin#392] [Ideas]
浏览了[demo](https://admin.spicyboy.cn/)加载挺快的，界面也好看，封装的各种功能也很实用，使用文档也相当完善。就是可惜老项目升级的话，js => ts  学习成本又增加了，真希望有个js版本

![image](https://github.com/HalseySpicy/Geeker-Admin/assets/43592730/233174d7-ed9c-4127-9bc3-e1d7aefcb8a5)

**Comment by tlerbao** [comment]
一个人撸项目的我，上了ts以后身心疲惫哈哈，多写多少代码....

**Reply by elarbor** [reply]
有没有配套的后端啊？

**Comment by zlecheng** [comment]
似乎都没找到相关js版本的代码
